docs/coding-guidelines/libraries-packaging.md

```diff
@@ -92,6 +92,3 @@ In order to mitigate design-time/build-time performance issues with source gener
     <DisableSourceGeneratorPropertyName>CustomPropertyName</DisableSourceGeneratorPropertyName> 
   </PropertyGroup>
 ```
```

Comment:
Should we add something to the docs around where the `-runtime` suffix is supported?  Does the infra fail in a sensible way when it's misused (if not, should we explicitly error)?

Reply:
After https://github.com/dotnet/arcade/pull/8418 merged and consumed, NuGet will throw during restore if a non .NETCoreApp tfm contains a suffix. It makes sense to add section to the [project guidelines](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/project-guidelines.md) clarifying which tfms can be runtime specific and how to apply conditions to specific runtimes correctly.

I plan to update that document with https://github.com/dotnet/runtime/pull/64500.